Makefile

```diff
@@ -483,7 +492,7 @@ endif
 
 ifeq ($(TARGET_PLATFORM),win32)
 electron-installer-zip: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
-electron-installer-nsis: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH).exe
+electron-installer-nsis: $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH).exe
```

Comment:
Hmmm, and does leaving the file in `dist/` instead of `dist/out/` mean that the snapshot-build publishing step will be broken?